test/GlobalDescriptor.ts

```diff
@@ -0,0 +1,276 @@
+import * as truffleTypes from '../types/truffle-contracts';

```

Comment:
Run formatter on this file, there are indents that are off.

Reply:
Actually, make it global-descriptor.ts, I messed up on my naming of userDescriptor.ts. Should be hyphentated.